Formula/gismo.rb

```diff
@@ -0,0 +1,72 @@
+class Gismo < Formula
+  desc "C++ library for isogeometric analysis (IGA)"
+  homepage "https://gismo.github.io"
+  url "https://github.com/gismo/gismo/archive/refs/tags/v21.12.0.tar.gz"
+  sha256 "4001b4c49661ca8b71baf915e773341e115d154077bef218433a3c1d72ee4f0c"
+  license "MPL-2.0"
+
+  head do
+    url "https://github.com/gismo/gismo.git", branch: "stable"
+
+    depends_on "cmake" => :build
+  end
+
+  depends_on "cmake" => :build
+  depends_on "openblas"
+  depends_on "suite-sparse"
+  depends_on "superlu"
+
+  on_macos do
+    depends_on "libomp"
```

Comment:
Why?

Reply:
@SMillerDev Thanks for taking the time to look into my PR. This is my first homebrew formula. What do you mean by 'why?'. Why the dependency on `superlu` or `libomp`? 

G+Smo can be built with SuperLU support. That's done in
```
args << "-DGISMO_WITH_SUPERLU=ON"
```
 The
```
on_macos do
  depends_on "libomp"
end
```
I found in the `torch vision.rb` formula.

Reply:
Why does it need libomp?

Reply:
G+Smo is a scientific geometry modelling and simulation package that performs quite 'heavy' computations. The core routines are parallelised with OpenMP. In my understanding, `libomp` is the way to enable OpenMP support. Is there another way?